controller/linodeobjectstoragebucket_controller.go

```diff
@@ -233,16 +162,10 @@ func (r *LinodeObjectStorageBucketReconciler) reconcileApply(ctx context.Context
 	return nil
 }
 
-func (r *LinodeObjectStorageBucketReconciler) reconcileDelete(ctx context.Context, bScope *scope.ObjectStorageBucketScope) error {
+// ctx has been removed from the param list
+func (r *LinodeObjectStorageBucketReconciler) reconcileDelete(bScope *scope.ObjectStorageBucketScope) error {
```

Comment:
We should be able to get rid of this function entirely now since we only added the finalizer to make sure we handled key revocation. This also means that we can get rid of the finalizer for OBJ buckets.